http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/Boot.scala

```diff
@@ -47,119 +50,142 @@ import scala.concurrent.duration._
+              leonardoService,
+              appDependencies.clusterHelper
+            )
+          )
+          system.actorOf(
+            ZombieClusterMonitor.props(zombieClusterMonitorConfig,
+                                       appDependencies.googleDataprocDAO,
+                                       googleProjectDAO,
+                                       appDependencies.dbReference)
           )
-        )
-        system.actorOf(
-          ZombieClusterMonitor.props(zombieClusterMonitorConfig,
+          system.actorOf(
+            ClusterToolMonitor.props(clusterToolMonitorConfig,
                                      appDependencies.googleDataprocDAO,
                                      googleProjectDAO,
-                                     appDependencies.dbReference)
-        )
-        system.actorOf(
-          ClusterToolMonitor.props(clusterToolMonitorConfig,
-                                   appDependencies.googleDataprocDAO,
-                                   googleProjectDAO,
-                                   appDependencies.dbReference,
-                                   appDependencies.metrics)
-        )
-      }
-      val clusterDateAccessedActor =
-        system.actorOf(ClusterDateAccessedActor.props(autoFreezeConfig, appDependencies.dbReference))
-      val proxyService = new ProxyService(proxyConfig,
-                                          appDependencies.googleDataprocDAO,
-                                          appDependencies.dbReference,
-                                          appDependencies.clusterDnsCache,
-                                          authProvider,
-                                          clusterDateAccessedActor)
-      val statusService = new StatusService(appDependencies.googleDataprocDAO,
-                                            appDependencies.samDAO,
-                                            appDependencies.dbReference,
-                                            dataprocConfig)
-      val leoRoutes = new LeoRoutes(leonardoService, proxyService, statusService, swaggerConfig)
-      with StandardUserInfoDirectives
-      IO.fromFuture(
-        IO(
-          Http()
-            .bindAndHandle(leoRoutes.route, "0.0.0.0", 8080)
-            .recover {
-              case t: Throwable =>
-                logger.error("FATAL - failure starting http server", t)
-                throw t
+                                     appDependencies.dbReference,
+                                     appDependencies.metrics)
+          )
+
+          // TODO Template and get the group and domain values from config
+          val dpImageUserGoogleGroupName = "kyuksel-test-dataproc-image-group-11"
+          val dpImageUserGoogleGroupEmail = WorkbenchEmail(s"$dpImageUserGoogleGroupName@test.firecloud.org")
+          val ch = appDependencies.clusterHelper
+          ch.createDataprocImageUserGoogleGroupIfItDoesntExist(appDependencies.googleDirectoryDAO,
```

Comment:
Could we pass `googleDirectoryDAO` in the constructor for `ClusterHelper` instead?